core-bundle/src/Intl/Countries.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Intl;
+
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\System;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\Intl\Countries as SymfonyCountries;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class Countries
+{
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    /**
+     * @var RequestStack
+     */
+    private $requestStack;
+
+    /**
+     * @var ContaoFramework
+     */
+    private $contaoFramework;
+
+    /**
+     * @var string
+     */
+    private $defaultLocale;
+
+    /**
+     * @var array<string>
+     */
+    private $countries;
+
+    public function __construct(TranslatorInterface $translator, RequestStack $requestStack, ContaoFramework $contaoFramework, array $availableCountries, string $defaultLocale, array $countriesList)
```

Comment:
Just a minor nitpick: `array $availableCountries, array $countriesList, string $defaultLocale` would be more logical. 🙈

Reply:
Fixed in 6b26513356bbe038839398a9823a7259669695e1